go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/ONSdigital/dp-frontend-search-controller
 
-go 1.21
+go 1.22.1
```

Comment:
I don't believe we usually specify in the go.mod the minor version. Can sometimes create problems.

Reply:
I did this to try and get the CI lint job to build but didn't try updating the linter first, so will revert and update the linter and see if I get any further 👍